core/app/modules/site-editor/assets/js/module.js

```diff
@@ -161,6 +161,21 @@ export default class SiteEditor {
 					video_url: 'https://www.youtube.com/embed/ACCNp9tBMQg',
 				},
 			},
-			];
+			{
+				type: 'loop',
+				icon: 'eicon-error-404', // TODO
+				title: __( 'Loop', 'elementor' ),
+				urls: {
+					thumbnail: elementorAppConfig.assets_url + '/images/app/site-editor/loop.svg',
+				},
+				tooltip_data: {
+					title: __( 'What is a loop?', 'elementor' ),
+					content: __( 'A Loop is a layout you can customize to display recurring dynamic content - like listings, posts, portfolios, products, , etc.', 'elementor' ),
+					tip: __( 'Start by creating a master item. All the other instances in the grid will match this design. Then go back to the widget in the editor panel and assign both a template and a source of content. Your grid should populate automatically.', 'elementor' ),
+					docs: 'https://go.elementor.com/app-theme-builder-loop',
+					video_url: '', // TODO
```

Comment:
we don't have a video, please remove the TODO

Reply:
removed